solr/distribution/build.gradle

```diff
@@ -50,40 +50,30 @@ if (project.ext.useGpgForSigning) {
 // Prepare the "source" distribution artifact.
 apply from: buildscript.sourceFile.toPath().resolveSibling("source-release.gradle")
 
-// Prepare the "binary" distribution artifact.
-apply from: buildscript.sourceFile.toPath().resolveSibling("binary-release.gradle")
-
-// Configure artifact signing.
-apply from: buildscript.sourceFile.toPath().resolveSibling("artifact-signing.gradle")
-
 
 // Set up the HTML-rendered "changes" distribution artifact by linking to documentation's output.
 configurations {
   changesHtml
-  solrTgz
-  solrTgzSignature
   docker
 }
 
-artifacts {
```

Comment:
do we want to keep the signature as an artifact?

Reply:
Ok I didn't realize that the signing plugin puts it into the original directory. Kind of complicates trying to rely on the output of the build, since the configuration we would be making for it here, lives in the build directory of `solr/packaging`.... Still need to work this out.